apiserver/firestoreutil/testutil.go

```diff
@@ -15,10 +15,9 @@
 package firestoreutil
 
 import (
+	"cloud.google.com/go/firestore"
 	"context"
 	"testing"
-
-	firestore "cloud.google.com/go/firestore"
```

Comment:
don't change these. external imports are typically in separate blocks. see other files.